app/src/components/market/outcome_table/index.tsx

```diff
@@ -162,20 +185,16 @@ export const OutcomeTable = (props: Props) => {
         ) : (
           <TDStyled textAlign={TableCellsAlign[3]}>{formatBigNumber(shares, collateral.decimals)}</TDStyled>
         )}
-        {disabledColumns.includes(OutcomeTableValue.Payout) ? null : withWinningOutcome ? (
-          <TDStyled textAlign={TableCellsAlign[4]}>{formatBigNumber(shares, collateral.decimals)}</TDStyled>
+        {disabledColumns.includes(OutcomeTableValue.Payout) ? null : withWinningOutcome && payouts ? (
+          <TDStyled textAlign={TableCellsAlign[4]}>{payouts[outcomeIndex]}</TDStyled>
         ) : (
           <TDStyled textAlign={TableCellsAlign[4]}>{formatBigNumber(shares, collateral.decimals)}</TDStyled>
```

Comment:
I'm not sure, but without a winningOutcome I think line 191 should be null.

Reply:
Yep, agree.